ios-target/View/Home/HomeViewController.swift

```diff
@@ -7,40 +7,80 @@
 //
 
 import UIKit
+import MapKit
 
 class HomeViewController: UIViewController {
   
   // MARK: - Outlets
   
-  @IBOutlet weak var welcomeLabel: UILabel!
-  @IBOutlet weak var logOut: UIButton!
+  @IBOutlet weak var mapView: MKMapView!
   
   var viewModel = HomeViewModel()
   
   // MARK: - Lifecycle Events
+  
   override func viewDidLoad() {
     super.viewDidLoad()
     viewModel.delegate = self
-    logOut.setRoundBorders(22)
+    setLocationServices()
+    viewModel.locationManager.checkLocationServices()
+    setLocationServices()
+    //TODO:
+//    viewModel.loadTargetPoints()
   }
   
   // MARK: - Actions
-  
-  @IBAction func tapOnGetMyProfile(_ sender: Any) {
-    viewModel.loadUserProfile()
-  }
 
-  @IBAction func tapOnLogOutButton(_ sender: Any) {
+  @IBAction func tapOnAddTarget(_ sender: Any) {
+    //targetFrom.showTargetForm()
+  }
+  
+  @IBAction func tapOnLogoutButton(_ sender: Any) {
     viewModel.logoutUser()
   }
+  
+  func setLocationServices() {
```

Comment:
This method looks like it should go on the ViewModel, you shouldn't interact with the manager directly from the view